block_test.go

```diff
@@ -73,3 +76,44 @@ func createEmptyBlock(t *testing.T, dir string, meta *BlockMeta) *Block {
 	testutil.Ok(t, err)
 	return b
 }
+
+// createPopulatedBlock creates a block with nSeries series, and nSamples samples.
+func createPopulatedBlock(tb testing.TB, dir string, nSeries, nSamples int) *Block {
+	head, err := NewHead(nil, nil, nil, 2*60*60*1000)
+	testutil.Ok(tb, err)
+	defer head.Close()
+
+	lbls, err := labels.ReadLabels("testdata/20kseries.json", nSeries)
```

Comment:
```suggestion
	lbls, err := labels.ReadLabels(filepath.Join("testdata","20kseries.json", nSeries)
```
otherwise these won't work on windows.

Reply:
👍